optimum/utils/input_generators.py

```diff
@@ -364,13 +416,23 @@ def __init__(
         else:
             self.num_choices = num_choices
 
-    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+    def generate(
+        self,
+        input_name: str,
+        framework: str = "pt",
+        int_dtype: str = "int64",
+        float_dtype: str = "fp32",
+        padding_side: str = "right",
```

Comment:
I'd rather have this `padding_side` in the init in order to keep the same signature for all the `generate` methods, but it does not matter too much